docs/querying/api.md

```diff
@@ -381,3 +381,73 @@ $ curl http://localhost:9090/api/v1/alertmanagers
   }
 }
 ```
+
+
+## TSDB Admin APIs
+These are APIs that expose database functionalities for the advanced user. These APIs are not enabled unless the `--web.enable-admin-api` is set.
+
+We also expose a gRPC API whose definition can be found [here](https://github.com/prometheus/prometheus/blob/master/prompb/rpc.proto). This is experimental and might change in the future.
+
+### Snapshot
+Snapshot creates a snapshot of all current data into `snapshots/<datetime>-<rand>` under the TSDB's data directory and returns the directory as response.
+
+*New in v2.1*
+
+```
+POST /api/v1/admin/tsdb/snapshot
+```
+
+```json
+$ curl -XPOST http://localhost:9090/api/v1/admin/tsdb/snapshot
+{
+  "name": "2017-11-30T15:31:59Z-2366f0a55106d6e1"
```

Comment:
This should be using the response envelope defined at the top of the file.

Reply:
Done.